packages/expo-modules-core/ios/Core/AppContext.swift

```diff
@@ -394,13 +394,13 @@ public final class AppContext: NSObject {
     // Install the modules host object as the `global.expo.modules`.
     EXJavaScriptRuntimeManager.installExpoModulesHostObject(self)
 
+    // Install `global.expo.EventEmitter`.
+    EXJavaScriptRuntimeManager.installEventEmitterClass(runtime)
```

Comment:
I had to change the order of installing the bindings, so that EventEmitter is already defined when we create SharedObject class